libpod/events.go

```diff
@@ -19,19 +19,32 @@ func (r *Runtime) newEventer() (events.Eventer, error) {
 	return events.NewEventer(options)
 }
 
-// newContainerEvent creates a new event based on a container
-func (c *Container) newContainerEvent(status events.Status) {
+func createContainerEvent(c *Container, status events.Status) events.Event {
```

Comment:
Why not hang this off Container? It appears you always call it from a method already hung off container.

Reply:
i dont really want this called directly. it never actually logs the event.

Reply:
No longer applicable